app/models/solidus_bolt/payment_method.rb

```diff
@@ -2,9 +2,6 @@
 
 module SolidusBolt
   class PaymentMethod < SolidusSupport.payment_method_parent_class
-    preference :bolt_api_key, :string
-    preference :bolt_signing_secret, :string
-
```

Comment:
Why you have removed these preferences here?

Reply:
well, we don't gather this data from the payment_method anymore, but directly from bolt_configuration

Reply:
I think we should keep them here. We don't know how the customer will use this extension. Maybe he will want to set more than one payment method for Bolt.